gflib/sprite.c

```diff
@@ -439,7 +439,11 @@ void SortSprites(void)
             }
         }
 
+#ifndef UBFIX
         while (j > 0
+#else
+        while (j > 1
```

Comment:
This introduces a bug where the first two elements won't be sorted properly because it will terminate the while loop too early.

Reply:
What about now?

Reply:
It's now buggier and is also missing the closing directive for the new UBFIX section.  I'm going to close this PR because it's clear you aren't testing or validating any of the changes, which is especially important for an important piece of code like this.

Reply:
Alright. I’m going to do an investigation and debug this section. Turns out this code is more complicated than I thought.

I apologize for my rash commits.